### ISSUE TITLE
not wrote profile any more。

### ISSUE THREAD

**Issue by guotao** [poad/apple-provisioning-profiles-downloader#1435]
Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

and then not Wrote IOS_APP_DEVELOPMENT profile  any more。。

**Comment by sudarshann** [comment]
I get the below 

```
Run poad/apple-provisioning-profiles-downloader@v1.0.6
bundle-id: ***
api-key-id: ***
issuer-id: ***
profile-type: IOS_APP_STORE
api-private-key: specified
api-private-key-file: 
token-duration: 
3 profiles found.
Wrote IOS_APP_STORE profile ***
Wrote IOS_APP_STORE profile ***
Wrote IOS_APP_STORE profile ***

Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-[11](https://github.com/QR-Transactions-Inc/app/actions/runs/4236446102/jobs/7361274108#step:3:12)-github-actions-deprecating-save-state-and-set-output-commands/
```


Then during the build stage I get 

```
/Users/runner/work/app/app/platforms/ios/***.xcodeproj: error: There are no accounts registered with Xcode. Add your developer account to Xcode (in target '***' from project '***')
xcodebuild: Command failed with exit code 65
/Users/runner/work/app/app/platforms/ios/***.xcodeproj: error: No profiles for 'com.***.***' were found: Xcode couldn't find any iOS App Development provisioning profiles matching 'com.***.***'. (in target '***' from project '***')

```

**Comment by poad** [comment]
@guotao @sudarshann 

Please update to v1.0.8.
It should be fixed to output without `set-output`.

**Comment by sudarshann** [comment]
Thanks @poad. 1.0.8 fixed set-output. But still I get the error with no accounts registered with xcode and no profiles. Any idea how that can be fixed?

**Comment by sudarshann** [comment]
I use ionic framework and the below is the build command and yes it does have allowProvisioningUpdates

/Applications/Xcode_13.2.1.app/Contents/Developer/usr/bin/xcodebuild -workspace ***.xcworkspace -scheme *** -configuration Release -destination generic/platform=iOS -archivePath ***.xcarchive -allowProvisioningUpdates archive CONFIGURATION_BUILD_DIR=/Users/runner/work/app/app/platforms/ios/build/device SHARED_PRECOMPS_DIR=/Users/runner/work/app/app/platforms/ios/build/sharedpch "EMBEDDED_CONTENT_CONTAINS_SWIFT = YES" ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES=NO "LD_RUNPATH_SEARCH_PATHS = \"@executable_path/Frameworks\""

**Comment by sudarshann** [comment]
@poad  If had found the issue. It seems your action is doing its job perfectly of downloading the provision profiles. I had not downlowed the certificates. I ended up following this doc 
https://docs.github.com/en/actions/deployment/deploying-xcode-applications/installing-an-apple-certificate-on-macos-runners-for-xcode-development

I had provided the provisioning profile manually and certifcate and it worked. Any way thanks your action worked good. in future  I will try to integrate your action for provision profile and the github reference only for certifcates when I get more time on this. 

Any way good job with our action.
